java/restheart/config.yaml

```
@@ -5,17 +5,8 @@ framework:
 build:
   - mvn package
 
-environment:
-  RHO: >
-    /http-listener->{\"enabled\": true, \"host\": \"0.0.0.0\", \"port\": 3000};
-    /core/name->\"restheart-benchmarker\";
-    /core/worker-threads->0;
-    /core/requests-limit->100000;
-    /fullAuthorizer/enabled->true;
-    /logging/log-level->\"OFF\";
-    /ping/enabled->false;
-
 binaries:
   - target/benchmark.jar
+  - etc/conf-overrides.yml
```

sorry, this not very obvious, but this should be upload in all containers

```suggestion
  - etc/conf-overrides.yml
  
files:
  - etc/conf-overrides.yml
```